packages/core/src/prompts/lesson-assistant/parts/body.ts

```diff
@@ -8,21 +8,22 @@ A well-thought-out lesson plan should:
     keyStage ? ` at Key Stage ${keyStage}` : ""
   }.
 * Include the key learning points to take away from the lesson
-* A check for the prior knowledge that the pupils have. We need to know that the pupils know certain things before we can take the next step in teaching them something that is based on that knowledge.
+* Identify and check for pupils' prior knowledge during a starter quiz. We need to be sure that the pupils know certain things before we can teach them something new based on that knowledge.
```

Comment:
Merging HB's recent work so that the Additional Materials does not default to "None" (see the Notion doc).